src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSession.cs

```diff
@@ -85,7 +88,14 @@ public async Task StartSessionAsync(bool runningUnderDebugger, CancellationToken
 
             HotReloadAgentFlags flags = runningUnderDebugger ? HotReloadAgentFlags.IsDebuggedProcess : HotReloadAgentFlags.None;
             await _hotReloadAgentManagerClient.Value.AgentStartedAsync(this, flags, cancellationToken);
-            WriteToOutputWindow(VSResources.HotReloadStartSession);
+            WriteToOutputWindow(
+                new HotReloadLogMessage(
+                    HotReloadVerbosity.Minimal,
+                    VSResources.HotReloadStartSession,
+                    Name,
+                    _variant.ToString()
```

Comment:
We call `ToString` on this int many times. Perhaps just store it as a string to avoid allocating on every call.